src/Reporter/GitlabReporter.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfTwig\Twigcs\Reporter;
+
+use FriendsOfTwig\Twigcs\Validator\Violation;
+use JsonException;
+use Symfony\Component\Console\Output\OutputInterface;
+use function hash;
+use function implode;
+use function json_encode;
+
+class GitlabReporter implements ReporterInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @param Violation[] $violations
+     *
```

Comment:
```suggestion
```

Reply:
@frankdekker 

Can we produce a scenario where a `JsonException` would be thrown? If not, perhaps we can remove the DocBlock entirely?

Reply:
I think I can, want me to add a test for it?

Reply:
Added test with JsonException. Let me know if you agree.